### ISSUE TITLE
Next update?

### ISSUE THREAD

**Issue by I-794** [BrenStro/SignMaker#7]
Will there ever be another update to this site?

**Comment by samuelh2005** [comment]
Yes, the successor project exists https://github.com/JKPotato-Computer/SignMaker.
